pkgs/by-name/ce/celeste-classic/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+, autoPatchelfHook
+, SDL2
+, practiceMod ? false
+}:
+
+let
+  directory = if practiceMod then "CELESTE*Practice*" else "CELESTE";
+  srcbin = if practiceMod then "celeste_practice_mod" else "celeste";
+  outbin = if practiceMod then "celeste-classic-pm" else "celeste-classic";
+in
+stdenvNoCC.mkDerivation {
+  pname = outbin;
+  version = "1.0";
+
+  src = fetchzip {
+    url = "https://www.speedrun.com/static/resource/174ye.zip";
+    hash = "sha256-GANHqKB0N905QJOLaePKWkUuPl9UlL1iqvkMMvw/CC8=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+  ];
+
+  buildInputs = [ SDL2 ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dsm755 ${directory}/${srcbin} $out/bin/${outbin}
+    install -Dm444 ${directory}/data.pod $out/bin/data.pod
```

Comment:
This puts `data.pod` to `bin`, while it isn't a binary. Instead, do something like this:

```patch
-    install -Dsm755 ${directory}/${srcbin} $out/bin/${outbin}
-    install -Dm444 ${directory}/data.pod $out/bin/data.pod
+    install -Dsm755 ${directory}/${srcbin} $out/lib/${outbin}/${outbin}
+    install -Dm444 ${directory}/data.pod $out/lib/${outbin}/data.pod
+    mkdir -p $out/bin
+    ln -s $out/lib/${outbin}/${outbin} $out/bin/
```